Extension/src/LanguageServer/ui.ts

```diff
@@ -110,12 +110,15 @@ export class UI {
 
     public activeDocumentChanged(): void {
         let activeEditor: vscode.TextEditor = vscode.window.activeTextEditor;
-        let show: boolean = (activeEditor && (activeEditor.document.languageId === "cpp" || activeEditor.document.languageId === "c"));
+        let isCpp: boolean = (activeEditor && ((activeEditor.document.languageId === "cpp" || activeEditor.document.languageId === "c")));
```

Comment:
Why add the extra parentheses?

Reply:
Oops, that's stale code -- also I just noticed a missing parenthesis in the isSettingsJson so I'll fix that.